runtime/win32.c

```diff
@@ -1258,3 +1258,47 @@ value caml_win32_xdg_defaults(void)
 
   CAMLreturn(result);
 }
+
+#include <winternl.h>
+
+typedef NTSTATUS (NTAPI *RtlDosLongPathNameToNtPathName_U_t)
+  (PCWSTR DosName, PUNICODE_STRING NtName, PWSTR *FilePart OPTIONAL, PVOID Reserved);
+
+static RtlDosLongPathNameToNtPathName_U_t RtlDosLongPathNameToNtPathName_U = NULL;
```

Comment:
```suggestion
static RtlDosLongPathNameToNtPathName_U_t RtlDosLongPathNameToNtPathName_U;
```

It's a static variable, initialized to `NULL` by default.